integration-test/1273-roads-access.py

```diff
@@ -0,0 +1,24 @@
+# Add surface properties to roads layer (at max zooms)
+
+# restricted access road in military base, Kraków, Poland
+# https://www.openstreetmap.org/way/43322699
+test.assert_has_feature(
+    18, 145574, 88813, 'roads',
```

Comment:
As I understand it will zoom out - is there any reason to expect that data may be dropped on zooming in?